### PR TITLE
Add possibility to sort tasks

### PR DESCRIPTION
**Pull request by PhilippMatthes**
Implement #238

### CODE REVIEW

**Comment by martinmo** [comment]
Thanks, Philipp! I have attached some inline comments. There are two things missing:

- Obviously, I need to figure out why Travis doesn't report the status... :(
- The default sorting, which should be easy to set in the Task model

Did you test the sorting actually works?

File: inloop/tasks/models.py [hunk 1]
```diff
@@ -97,6 +97,14 @@ class Task(models.Model):
...
     objects = TaskQuerySet.as_manager()
 
+    @classmethod
+    def model_field_exists(cls, field):
+        try:
+            cls._meta.get_field(field)
+            return True
+        except models.FieldDoesNotExist:
+            return False
+
```
[the remark this answers is not in the document]
Comment (martinmo) [reply inloop/tasks/models.py]: For the time being it might be ok. Right now I also don't know a better solution.

**Comment by PhilippMatthes** [comment]
The sorting works, it also defaults to sort by `title`. This is done by
```python
order_by = request.GET.get('order_by', 'title')
```
Which checks, if there is a parameter `order_by` and otherwise defaults to `title`. Should this default be moved to the model?

**Comment by martinmo** [comment]
> Which checks, if there is a parameter `order_by` and otherwise defaults to `title`. Should this default be moved to the model?

Originally I wanted to the default sorting to be by pubdate and deadline. But pubdate is sufficient as a default. Yes, and the default should be set in the Meta class of the Task model.

**Comment by PhilippMatthes** [comment]
> > Which checks, if there is a parameter `order_by` and otherwise defaults to `title`. Should this default be moved to the model?
> 
> Originally I wanted to the default sorting to be by pubdate and deadline. But pubdate is sufficient as a default. Yes, and the default should be set in the Meta class of the Task model.

Ok, should be a quick fix.

**Pull request merged**